CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to the [Nucleus Python Client](https://github.com/scaleapi/n
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.14.20](https://github.com/scaleapi/nucleus-python-client/releases/tag/v0.14.20) - 2022-09-23
+
+### Fixed
+- Local uploads are correctly batched and prevents flooding the network with requests
```

Comment:
Any estimate for up to which number / size of data things would work robustly now? Or is there basically no limit and it would just be annoyingly slow?

Reply:
> Any estimate for up to which number / size of data things would work robustly now? 

IMO it should robustly work for any number of item uploads, since the client is now correctly batching.